SDL_ttf.h

```diff
@@ -678,6 +687,21 @@ extern DECLSPEC int SDLCALL TTF_GetFontKerning(const TTF_Font *font);
  */
 extern DECLSPEC void SDLCALL TTF_SetFontKerning(TTF_Font *font, int allowed);
 
+/**
+ * Query the font bounding box.
+ * 
+ * The bounding box defines bounds large enough to contain any glyph from
+ * the font. It is expressed in pixel offsets from glyph's origin (0,0),
+ * with Y axis pointing upwards. Thus ymax offset may be seen as the
+ * "maximum ascender" and ymin offset - as the "minimum descender".
+ *
+ * \param font the font to query.
+ * \param bbox the TTF_BBox struct to fill.
+ *
+ * \since This function is available since SDL_ttf 2.26.0.
+ */
+extern DECLSPEC void SDLCALL TTF_GetFontBBox(const TTF_Font *font, TTF_BBox *bbox);
```

Comment:
Rather than creating a new type, can you have separate pointers for each element that can be optionally filled in?

Reply:
Changed the prototype following `TTF_GlyphMetrics` as an example.